### ISSUE TITLE
Guide content is broken on IE 11

### ISSUE THREAD

**Issue by anukandimalla** [cityofaustin/techstack#2892]
Bug Description: The Guide body content is pretty broken on IE 11. Shows up super narrow with a scroll bar in the middle. Table of contents doesn't appear.

URL: https://janis.austintexas.io/en/health-safety/healthcare-prevention/disease-prevention/mobile-food-permitting-process-guide/

![image](https://user-images.githubusercontent.com/5839508/64298400-06771b80-cf3b-11e9-8479-065c56b7401d.png)

To Reproduce:

Open the link in IE 11
2.Scroll down to the Guide Body
Tested the screen Resolution at 1920x969 1024x768 800x600 currently working on the mobile(Galaxy 7)
Current behavior: The content body width is too narrow. Please see the attached screenshot for behavior. Reproduced Bug.png

Expected behavior: The body content should be the viewport size of 1440x900
Screen Shot 2019-09-04 at 10.33.47 AM.png

Tested on Windows 7 - IE11

**Comment by anukandimalla** [comment]
Screenshots for images are missing on the report and please find the screenshots below
<img width="1904" alt="Reproduced Bug" src="https://user-images.githubusercontent.com/52670306/64271095-584f7f80-cf02-11e9-818a-4c95cf26f948.png">

**Comment by robbobfrh84** [comment]
fixed with https://github.com/cityofaustin/janis/pull/524 😄

**Comment by Niicck** [comment]
Merged into staging. QAable at  https://janis.austintexas.io/en/health-safety/healthcare-prevention/disease-prevention/mobile-food-permitting-process-guide/

**Comment by desigonz** [comment]
This looks amazing! I filed a new bug for IE11 step number display (#3039), but that's outside of the scope of guides. QA complete and good to go to production.

**Comment by robbobfrh84** [comment]
@desi, assigned myself to 👉 #3039

**Comment by desigonz** [comment]
Hey @robbobfrh84—no need to work on #3039 this sprint, since it's unrelated to the guide content bug (just something I found and logged along the way). You can keep focusing on your sprint goal issues and then we can evaluate what you should pick up.

**Comment by briaguya** [comment]
looks like this has been in prod since noon yesterday:
@robbobfrh84 - have you done a quick check to make sure everything is working as expected in prod?
@desigonz is there anything else that needs to happen before we can close this out?

**Comment by desigonz** [comment]
@briaguya This one is good to close on my end. Not sure we can really check this one on prod because we haven't published a guide yet there.

**Comment by desigonz** [comment]
But it looked good in staging
